examples/align-gl/CMakeLists.txt

```diff
@@ -10,7 +10,8 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS 1)
 if(BUILD_GRAPHICAL_EXAMPLES AND NOT APPLE)
 
 add_executable( ${PROJECT_NAME} rs-align-gl.cpp
-    ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp
+    ../../third-party/imgui/imgui.cpp ../../third-party/imgui/imgui_draw.cpp ../../third-party/imgui/imgui_impl_glfw.cpp 
```

Comment:
Is this happens for all examples & tool maybe we wand a helper cmake file with a variable for IMGUI files?

then we can include it and use it's variable?

This way if we want to add/remove a file we touch 1 place and all examples and tools will get it

Reply:
I added this line in the tools and examples CMakeLists
file(GLOB IMGUI_SOURCES "../third-party/imgui/*.cpp")
and now all imgui cpp source files are stored in the IMGUI_SOURCES variable and can be used in the CMakelists of each tool and example if needed